packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -157,6 +159,44 @@ class StripeHelper {
     );
   }
 
+  /**
+   * Create a stripe customer
+   *
+   * @param {string} uid
+   * @param {string} email
+   * @param {string} displayName
+   * @param {string} paymentToken
+   *
+   * @returns {Promise<Customer>}
+   */
+  async createCustomer(uid, email, displayName, paymentToken) {
+    const customer = await this.stripe.customers.create({
```

Comment:
Tiny nit: not doing any exception handling here, so could just return the result of the `create` call directly without an `await`. Same with `updateCustomerPaymentMethod` below. (Doesn't hurt anything to leave as-is, though.)

Reply:
I was thinking about doing that. The reason it is as it currently appears: I wanted to be able to set a breakpoint at the return so I could check the value.